### ISSUE TITLE
aot eager accuracy error: convit_base

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#1275]
Repro:
```
benchmarks/timm_models.py  --training --accuracy-aot-nop -d cuda --use-eval-mode  --amp --only convit_base

INCORRECT
```

**Comment by eellison** [comment]
Passes on master
